0000-Install-and-Test/NodeRED-and-TJBot-Nodes/soict_s4v/extinstall.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+cd /home/pi/Desktop/
+curl -O http://tjbot.techlinkvn.com/soict_s4v.zip
```

Comment:
add more curl link for node-red lab

Reply:
Too many links make the upgrade uncontrollable. I think that we could have more files to download, cannot predictable and could lack URL, so zip all of them into only one file is simplier